addon/components/detail-instructors.hbs

```diff
@@ -38,12 +38,25 @@
         @availableInstructorGroups={{this.availableInstructorGroups}}
         @instructorGroups={{this.instructorGroupBuffer}}
         @instructors={{this.instructorBuffer}}
+        @showDefaultNotLoaded={{false}}
       />
     {{else}}
-      <DetailInstructorsList
-        @instructorGroups={{await @ilmSession.instructorGroups}}
-        @instructors={{await @ilmSession.instructors}}
-      />
+      {{#if this.instructorCount}}
+        <SelectedInstructors
+          @instructors={{await @ilmSession.instructors}}
```

Comment:
I don't think the `await` is necessary here. The value of `ilmSession.instructors` from Ember Data model is a proxy that will resolve to an array that has the values. We've been using `await` on these as we mixed some internal promises and some ember-data stuff, but I think we can move away from it nearly everywhere now.

Reply:
:heavy_check_mark: